frontend/src/utils/projectPage.js

```diff
@@ -52,18 +104,88 @@ export const getKitspaceYAMLJson = async assetsPath => {
   return [res.ok, res.ok ? await res.json() : kitspaceYAMLFields]
 }
 
+export const processedAssets = async assetsPath => {
+  const statusPath = getStatusPathFrom(assetsPath)
+  const rootStatusPath = statusPath.replace(/HEAD.+/, 'HEAD/')
+  const rootAssetsPath = assetsPath.replace(/HEAD.+/, 'HEAD/')
+
+  // https://github.com/kitspace/kitspace-v2/tree/master/processor#parameters
+  const assetsNames = [
+    'gerber-info.json',
+    'images/bottom.svg',
+    'images/top.svg',
+    'images/top.png',
+    'images/top-large.png',
+    'images/top-meta.png',
+    'images/top-with-background.png',
+    // 'images/layout.svg',
+    '1-click-BOM.tsv',
+    'bom-info.json',
+    'interactive_bom.json',
+    // 'kitspace-yaml.json',
```

Comment:
No reason for that to be there commented out I think.

Reply:
Resolved in 262253c.